1-exercises/B-object-arrays/exercise-1.js

```diff
@@ -13,7 +13,7 @@ let people = [
   { name: "Elise", age: 35 },
 ];
 
-let peopleNames =
+let peopleNames = people.map(people=>people.name);
```

Comment:
This is a good use of the map() function. And obviously the code is simple and functional, so its a thumbs up from me.